packages/gatsby/src/redux/actions/public.js

```diff
@@ -1478,4 +1479,50 @@ actions.unstable_createNodeManifest = (
   }
 }
 
+/**
+ * Stores request headers for a given domain to be later used when making requests for Image CDN (and potentially other features).
+ *
+ * @param {Object} $0
+ * @param {string} $0.domain The domain to store the headers for.
+ * @param {Object} $0.headers The headers to store.
+ */
+actions.setRequestHeaders = ({ domain, headers }, plugin: Plugin) => {
```

Comment:
It does, I believe the actions become "bound"? Which I believe is just a wrapper being added around these actions that adds the plugin and also adds a dispatch so when the Gatsby site calls the action it fires it off instead of just returning an object.